lib/Settings/WhoHasAccessSettings.php

```diff
@@ -35,7 +37,16 @@ class WhoHasAccessSettings implements ISettings {
 	 * @return TemplateResponse
 	 */
 	public function getForm():TemplateResponse {
-		return new TemplateResponse('privacy', 'who-has-access');
+		$themingDefaults = OC::$server->getThemingDefaults();
```

Comment:
@juliushaertl fyi I've checked the server code and we actually already test if the theming app is enabled in the server container, hence this check here isn't necessary :)